wagtail/admin/rich_text/editors/hallo.py

```diff
@@ -104,18 +104,16 @@ def __init__(self, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
-    def translate_value(self, value):
+    def format_value(self, value):
         # Convert database rich text representation to the format required by
         # the input field
+        value = super().format_value(value)
+
         if value is None:
             return None
 
         return self.converter.from_database_format(value)
 
-    def render(self, name, value, attrs=None):
-        translated_value = self.translate_value(value)
-        return super().render(name, translated_value, attrs)
-
```

Comment:
I agree... I only left this out of #4716 because I abandoned that strand of work once it became clear that it wouldn't be necessary for Django 2.1 support. Happy to add it if you think it's worth doing for completeness.